patreon.php

```diff
@@ -67,6 +67,8 @@
 <br /><br />
 If you request that your data be deleted from this website, this data will also be deleted and Patreon functionality will not work. You would need to register on this website and log in to this website with Patreon again in order to re-populate this data and have Patreon functionality working again.' );
 
+define( "PATREON_DIRECT_UNLOCKS_NOT_ON", 'Direct unlocks need to be turned on for using this feature. Please refer to developer documentation. If you see this message, please inform site administration.' );
```

Comment:
In order to use this feature, direct unlocks need to be turned on. Please refer to the developer documentation. https://www.patreon.com/apps/wordpress